R/target_market_share.R

```diff
@@ -0,0 +1,195 @@
+#' Add targets for production, using the market share approach
+#'
+#' This function calculates the portfolio-level production targets, as
+#' calculated using either the technology market share ratio `tmsr` or sector
+#' market share percentage `smsp`.
+#'
+#' @param data A "data.frame" like the output of
+#'   [r2dii.match::prioritize()].
+#' @param ald An asset level dataframe like [r2dii.data::ald_demo].
+#' @param scenario A scenario dataframe like [r2dii.data::scenario_demo_2020].
+#' @param region_isos A dataframe like [r2dii.data::region_isos] (default).
+#' @param use_credit_limit Logical vector of length 1. `FALSE` defaults to using
+#'   the column `loan_size_outstanding`. Set to `TRUE` to use the column
+#'   `loan_size_credit_limit` instead.
+#' @param by_company Logical vector of length 1. `FALSE` defaults to outputting
+#' `weighted_production_value` at the portfolio-level. Set to `TRUE` to output
+#' `weighted_production_value` at the company-level.
+#'
+#' @return A tibble with the same groups as the input (if any) and columns:
+#'   `tmsr_target_weighted_production` and `smsp_target_weighted_production`.
```

Comment:
I think we no longer output these columns.

Reply:
Correct, updated this as well!